src/main/kotlin/org/partiql/scribe/sql/utils/ConversionUtils.kt

```diff
@@ -16,3 +28,104 @@ public fun RelConverter.RelContext.toQueryBodySFW(): QueryBody.SFW {
         having = this.having,
     )
 }
+
+// ////////////////////////////// AST -> Text utils
+
+internal fun unquotedStringExpr(s: String): Expr {
+    return exprVarRef(Identifier.regular(s), isQualified = false)
+}
+
+// returns the string alias of an expr if an alias can be inferred. returns null when an alias cannot be inferred
+// and must be generated
+internal fun Expr.inferredAlias(): String? {
```

Comment:
self-review: porting this PR's change to remove redundant `AS` aliases https://github.com/partiql/partiql-scribe/pull/70. Currently only is for Redshift. We should also port to Spark + Trino dialects at some point https://github.com/partiql/partiql-scribe/issues/71.